kernel/signal.c

```diff
@@ -213,9 +246,20 @@ static long _handle_one_signal(
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wstack-usage="
+
+int myst_signal_has_active_signals(myst_thread_t* thread)
+{
+    uint64_t unblocked = (~thread->signal.mask) |
+                         ((uint64_t)1 << (SIGKILL - 1)) |
+                         ((uint64_t)1 << (SIGSTOP - 1));
```

Comment:
Seems `((uint64_t)1 << (SIGKILL - 1)) | ((uint64_t)1 << (SIGSTOP - 1))` is used a lot. Maybe worth a `#define`

Reply:
done